### ISSUE TITLE
Preview mode in "inline editor" doesn't allow to see the whole content

### ISSUE THREAD

**Issue by zvirja** [k0shk0sh/FastHub#919]
**FastHub Version: 4.1.0**  
**Android Version: 7.0 (SDK: 24)**  
**Device Information:**  
- **Manufacturer:** samsung  
- **Brand:** samsung  
- **Model:** Galaxy Tab S2 8.0  
---

Open any issue/pr and go to the bottom. You will a see a text area to enter the comment. If you put a multi-line comment there and press the "preview" button, you will see a part of the comment only. It's impossible to scroll to see the whole comment. The only way to see whole comment would be to press the "full screen" button and later hit the "preview" button  again.

Basically, I see a few ways to solve the issue:
1. Allow to scroll in preview. It would be still inconvenient as control height is 3 lines only.
1. Enter the preview in full screen mode and return back correctly.
1. Put comment directly to timeline with some mark that it's a preview. It will allow to see how comment will actually look in context.

Given that currently the inline preview is non-usable, the either way would help, but the last one looks very attractive :wink:

Notice, issue is relevant for both multi-line and simply long comments.

**Comment by zvirja** [comment]
Could you add a link to duplicated issue when you close with duplicate? I'd advice to use the "Duplicate of" keyword as GitHub recognizes them.

**Comment by k0shk0sh** [comment]
#896
